### PR TITLE
Preparing for booster 0.6.0 release

### PR DESCRIPTION
**Pull request by ryan-summers**
This PR preps for a 0.6.0 release.

### CODE REVIEW

**Comment by jordens** [comment]
https://github.com/quartiq/booster/blob/cab97464f4e34e862371bb2fa25784dddfdc79a7/Cargo.toml#L67 needs to go first

**Comment by jordens** [comment]
And I guess some/many of the included crates in booster also need a bump (assuming they have changed, for e.g. the e-h 1 port).

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+None yet!
```
Comment (jordens) [inline CHANGELOG.md:9]: Only add this section after release.

**Comment by ryan-summers** [comment]
@jordens There's also the `debounced-pin` and `minireq` crates. We can publish an updated minireq, but `debounced-pin` isn't owned by us. We could:
1. Republish rust-debounced-pin (and maybe move it over to the Rust-embedded-community)
2. Move the implementation for debouncing directly into `booster` (it's not complex)
3. Find an alternative crate

Ay particular preference here?

**Review by jordens — COMMENTED** [review]
No releases for the `ad5627` etc. crates?

**Comment by ryan-summers** [comment]
They all have their versions already bumped. I ran through about 30 minutes ago and released them all as 0.2 already.

**Review by jordens — APPROVED** [review]

**Review by jordens — CHANGES_REQUESTED** [review]
> warning: package `cortex-m-rt v0.7.4` in Cargo.lock is yanked in registry `crates-io`, consider updating to a version that is not yanked

**Review by jordens — APPROVED** [review]

**Pull request merged**